datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/counter.py

```diff
@@ -236,13 +299,52 @@ def _configure_counters(self, available_counters, available_instances):
                         self.tags,
                         custom_transformers.get(counter_name),
                         tag_name,
+                        self.include_wildcards,
+                        self.duplicate_instances_exist,
+                        self,
                     )
 
         self.counters.extend(counters.values())
 
+    def _get_counters_type(self):
+        # Enumerate all counter to find if it is single or multiple instance counter
+        # The very virst iteration should be sufficienmt, just in case enumerate all
+        for i, entry in enumerate(self.counters_config, 1):
+            if not isinstance(entry, dict):
+                raise ConfigTypeError(
+                    f'Entry #{i} of option `counters` for performance object `{self.name}` must be a mapping'
+                )
+
+            for counter_name, _ in entry.items():
+                # https://docs.microsoft.com/en-us/windows/win32/api/pdh/nf-pdh-pdhvalidatepatha
+                # https://mhammond.github.io/pywin32/win32pdh__ValidatePath_meth.html
+
+                # Check for multi-instance counter path
+                possible_path = construct_counter_path(
+                    machine_name=self.connection.server,
+                    object_name=self.name,
+                    counter_name=counter_name,
+                    instance_name='*',
+                )
+                if win32pdh.ValidatePath(possible_path) == 0:
```

Comment:
Damn that is very good question/catch 👍 I do not think it will work as intended. I will test it, I think I have no choice but to use Add[English]Counter directly.

Reply:
Test it. It will not work if localized is true because the API is not localized aware. E.g English object on Russian Windows or reverse Russian object on English OS. The error is No Object is found. It is not a new flaw, still. 

Also I check again PdhAdd[English]Counter and it works as expected. 

Thank you @clarkb7